src/cli.rs

```diff
@@ -485,6 +485,10 @@ pub struct Misc {
     /// Defaults to the number of CPUs
     #[clap(env, long)]
     pub threads: Option<usize>,
+
+    /// Option to only serve HTTP instead for testing.
+    #[clap(env, long)]
+    pub insecure_serve_http_only: bool,
```

Comment:
Maybe this better belongs in `HttpServer` ?

Reply:
Moved it, but haven't prefixed it with `http_server`. If that is preferred, I can add it. I just found it a bit odd to then use `cli.http_server.http_server_insecure_serve_http_only`.

Reply:
Yes, it's odd, but sadly required for correct env var names generation. `clap` uses the var name for this, so this redundancy is needed.

Reply:
done